src/main/java/htsjdk/samtools/CRAMIterator.java

```diff
@@ -41,7 +41,6 @@
     private final CountingInputStream countingInputStream;
     private final CramHeader cramHeader;
     private final ArrayList<SAMRecord> records;
-    private SAMRecord nextRecord = null;
```

Comment:
This would resolve #1280 - but what was this *intended* to do?

Reply:
Probably cut&pasted from some other iterator, if I had to guess.